packages/flutter/test/material/search_test.dart

```diff
@@ -991,9 +991,30 @@ void main() {
     expect(find.text('HomeBody'), findsOneWidget);
     expect(find.text('HomeTitle'), findsOneWidget);
     expect(find.text('Suggestions'), findsNothing);
+
     expect(selectedResults, <String>['Result']);
   });
 
+  testWidgets('Leading width size is 16', (WidgetTester tester) async {
+    final _TestSearchDelegate delegate = _TestSearchDelegate();
```

Comment:
The `_TestSearchDelegate` should probably instantiated with a custom `leadingWidth` rather than a hard coded value inside of `_TestSearchDelegate`.

Reply:
yep, i've changed this logic. Thank you